avm/ptn/ai-platform/baseline/main.bicep

```diff
@@ -100,6 +100,7 @@ module storageAccount_privateDnsZones 'br/public:avm/res/network/private-dns-zon
     name: '${uniqueString(deployment().name, location, zone)}-storage-private-dns-zones'
     params: {
       name: zone
+      enableTelemetry: enableTelemetry
```

Comment:
Recommend to disable workflows before merging, and then running the changed modules manually after to publish